app/javascript/entrypoints/application.js

```diff
@@ -9,3 +9,10 @@ import '@hotwired/turbo-rails'
 // const channels = import.meta.globEager('./**/*_channel.js')
 
 import '~/controllers'
+
+import ahoy from 'ahoy.js'
+
+ahoy.trackView()
```

Comment:
Do we want to add this into a `turbo:load` event listener so it tracks it in the right way? Or did you do it like this on purpose?

Reply:
this is a good point, I used the install recommended by the documentation 
I will look into the source and do a few testing